ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -33,24 +33,36 @@ public typealias MSPersonaListView = PersonaListView
 
 @objc(MSFPersonaListView)
 open class PersonaListView: UITableView {
-    private enum Section: Int {
-        case personas
-        case searchDirectory
-    }
-
-    enum SearchDirectoryState {
+    /// SearchDirectory button state enum
+    public enum SearchDirectoryState {
         case idle
         case searching
         case displayingSearchResults
     }
 
+    private enum Section: Int {
+        case personas
+        case searchDirectory
+    }
+
     /// The personas to display in the list view
     @objc open var personaList: [Persona] = [] {
         didSet {
             reloadData()
         }
     }
 
+    /// searchDIrectoryState variable (persona list to reload rows on state change)
+    public var searchDirectoryState: SearchDirectoryState = .idle {
```

Comment:
>public [](start = 4, length = 6)

please add @objc to all public/open vars/methods

Reply:
Its not possible to do as SearchDirectoryState enum isn't @onjc. and compiler isn't allowing to create enums of type @objc as mentioned din another comment
<img width="1375" alt="image" src="https://user-images.githubusercontent.com/67058720/85665292-e0448700-b6d8-11ea-8879-89e7380bc94a.png">

Reply:
you can add "@objc(MSFSearchDirectoryState)" one live above the definition of the enum, instead of adding @objc next in the same line, and that should work